js/packages/teams-ai/src/AI.ts

```diff
@@ -474,17 +474,6 @@ export class AI<TState extends TurnState = TurnState> {
         start_time?: number,
         step_count?: number
     ): Promise<boolean> {
-        // Populate {{$temp.input}}
-        if (typeof state.temp.input != 'string') {
-            // Use the received activity text
-            state.temp.input = context.activity.text;
-        }
-
-        // Initialize {{$allOutputs}}
-        if (state.temp.actionOutputs == undefined) {
-            state.temp.actionOutputs = {};
-        }
-
```

Comment:
Why are these lines being removed?

Reply:
they're being initialized in the application class instead.